meiga/failures.py

```diff
@@ -8,15 +9,23 @@
 
 class WaitingForEarlyReturn(Error):
     result: "AnyResult"
+    called_from: Union[str, None]
 
     def __init__(self, result: "AnyResult") -> None:
         self.result = result
+        try:
+            stack = inspect.stack()[2]
```

Comment:
Por qué 2 y 3? Seguro que estos índices no cambian?

Reply:
From the doc: https://docs.python.org/3/library/inspect.html#inspect.stack

inspect.stack()[2] -> `filename`

And the when we get the FrameInfo, we use [3] to access [function](https://docs.python.org/3/library/inspect.html#inspect.FrameInfo.function)